src/core/server/integration_tests/saved_objects/jest.integration.config.js

```diff
@@ -16,9 +16,10 @@ module.exports = {
   preset: '@kbn/test/jest_integration',
   rootDir: '../../../../..',
   roots: [
-    '<rootDir>/src/core/server/integration_tests/saved_objects/routes',
-    '<rootDir>/src/core/server/integration_tests/saved_objects/service',
-    '<rootDir>/src/core/server/integration_tests/saved_objects/validation',
+    '<rootDir>/src/core/server/integration_tests/saved_objects/registration',
```

Comment:
Moved the type_registration to its own subfolder because it was getting "lost" within group3 in the migrations integration tests.